cyverse_allocation/views.py

```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
+
+# Create your views here.
```

Comment:
Structure comment: we may want to `cyverse_allocation` -> `allocation` and toss the existing allocation structure as part of this PR.